vignettes/data_outputs.Rmd

```diff
@@ -14,13 +14,11 @@ knitr::opts_chunk$set(
 )
 ```
 
-```{r setup, echo = FALSE}
+```{r setup, echo = FALSE, message = FALSE}
+library(pacta.loanbook)
+
 plot_table <- function(dataset_name) {
-  table <- dplyr::bind_rows(
-    r2dii.match::data_dictionary,
-    r2dii.analysis::data_dictionary,
-    r2dii.plot::data_dictionary
-  )
+  table <- data_dictionary
```

Comment:
Maybe we need to add some filter or explained bit more here. Previously we actually generated a data dictionary of outputs only, by omitting `r2dii.data::data_dictionary`. Now the new `pacta.loanbook::data_dictionary` will add that one as well. And that contains input descriptions

Reply:
this still works.... this is just code to load the `data_dictionary` that has the needed information (in this case, the new `pacta.loanbook::data_dictionary` contains everything), but the only tables printed on the the data outputs page are those that are explicitly called below, which is still only output datasets

https://rmi-pacta.github.io/pacta.loanbook/pr-preview/pr-82/articles/data_outputs.html#prep_trajectory-output